hubblestack/daemon.py

```diff
@@ -312,6 +313,15 @@ def run_function():
     else:
         print(ret)
 
+    if(__opts__['outfile']):
+        log.info('Writing output to file {0}'.format(__opts__['outfile']))
+        try:
+            outputfile = open(__opts__['outfile'], "w")
```

Comment:
Lets use with option rather than open.
Lets not dump info into log file as it's only for console.